charts/kured/templates/daemonset.yaml

```diff
@@ -81,6 +81,9 @@ spec:
           {{- if .Values.configuration.slackUsername }}
             - --slack-username={{ .Values.configuration.slackUsername }}
           {{- end }}
+          {{- if .Values.configuration.notifyURL }}
+            - --notify-url={{ .Values.configuration.notifyURL }}
+          {{- end }}
           {{- if .Values.configuration.messageTemplateDrain }}
```

Comment:
If we change the helm chart, we should probably bump its version.

Reply:
AFAICS it's already bumped from another PR, but not released yet, so we should be good.

Reply:
Oh that's interesting. I am not sure how chart-testing action works on the linting, does it bases itself on what has been already released, or on the existing code? If it's existing code, it means we can end up skipping releases.

@ckotzbauer do you think it's okay to do it this way, or should we edit the chart linting action? For me it kinda makes sense to bump the chart at the same time as we modify code, and release when we are ready, even if it means skipping a release.